cmd/install.go

```diff
@@ -0,0 +1,95 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+	"github.com/elastic/elastic-package/internal/packages"
+	"github.com/elastic/elastic-package/internal/packages/installer"
+)
+
+const installLongDescription = `Use this command to install/uninstall the package in Kibana.
+
+The command uses Kibana API to install/uninstall the package in Kibana. The package must be exposed via the Package Registry.
+
+Context:
+  package`
+
+func setupInstallCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install",
+		Short: "Install the package",
+		Long:  installLongDescription,
+		RunE:  installCommandAction,
+	}
+	cmd.Flags().BoolP(cobraext.UninstallPackageFlagName, "u", false, cobraext.UninstallPackageFlagDescription)
```

Comment:
It feels a little odd (to me) that we use `elastic-package install -u` to uninstall a package. I think it would be clearer to create an `elastic-package uninstall` command instead. Even in terms of implementation this would be cleaner IMO — for example, it doesn't really make sense to check any compatibility conditions when uninstalling a package.

Reply:
I agree with uninstall. 

Regarding the compatibility conditions... I need this feature somewhere in elastic-package, so that I can implement the support matrix between packages and stacks (https://github.com/elastic/integrations/issues/537#issuecomment-779253207). I planned to use the tool to check if the CI should spawn the stack, because the package should be compatible.

If you have other ideas for the support check or a better place to put it, feel free to share them.

Reply:
No no, I think the compatibility check the way you have it implemented for `elastic-package install` is just fine. I was saying it is not needed for `elastic-package uninstall`, that's all.

Reply:
This might helps clarify: I would remove the `-u` option from this PR and make it just about installing a package. Then put up a separate PR that implements `elastic-package uninstall` (and in this PR I think we don't need the compatibility checks).

Reply:
I removed uninstallation from this PR.